src/core/i18n/en/translation.en.json

```diff
@@ -512,16 +512,32 @@
         "OTHER": "Other"
       },
       "templateTypes": {
-        "WhiteboardTemplate": "Whiteboard Template",
-        "WhiteboardTemplate_plural": "Whiteboard Templates",
-        "PostTemplate": "Post Template",
-        "PostTemplate_plural": "Post Templates",
-        "InnovationFlowTemplate": "Innovation Flow",
-        "InnovationFlowTemplate_plural": "Innovation Flow Templates",
-        "CalloutTemplate": "Collaboration Tool Template",
-        "CollaborationToolTemplate_plural": "Collaboration Tool Templates",
-        "CommunityGuidelinesTemplate": "Community Guidelines Template",
-        "CommunityGuidelinesTemplate_plural": "Community Guidelines Templates"
+        "Whiteboard": "Whiteboard Template",
+        "Whiteboard_plural": "Whiteboard Templates",
+        "Post": "Post Template",
+        "Post_plural": "Post Templates",
+        "InnovationFlow": "Innovation Flow",
+        "InnovationFlow_plural": "Innovation Flow Templates",
+        "Callout": "Collaboration Tool Template",
+        "Callout_plural": "Collaboration Tool Templates",
+        "CommunityGuidelines": "Community Guidelines Template",
+        "CommunityGuidelines_plural": "Community Guidelines Templates",
+        "Collaboration": "Collaboration Template",
+        "Collaboration_plural": "Collaboration Template"
+      },
+      "templateType": {
```

Comment:
Do we need both `templateTypes` and `templateType` (different keys, same values)?